uuid_test.go

```diff
@@ -316,7 +316,7 @@ func TestNodeID(t *testing.T) {
 	}
 	node1 := NodeID()
 	if node1 == nil {
-		t.Errorf("NodeID nil after SetNodeInterface\n", s)
+		t.Error("NodeID nil after SetNodeInterface\n", s)
```

Comment:
Thanks Shawn!  Sorry for my delayed responses.  The newline should also be removed from the error message.  Thanks!